run.sh

```diff
@@ -1,30 +1,45 @@
 #!/bin/bash
-set -euxo pipefail
+set -euo pipefail
+
+function outputResult() {
+  local result=$1
+  local exitCode=$2
+
+  echo $result
+
+  if [ $OUTPUT_RESULT = true ]; then
+    echo "::set-output name=result::$result"
+  else
+    exit $exitCode
+  fi
+}
+
 if [ $GITHUB_EVENT_NAME = check_run ]
 then
   if [ "$(jq -r .check_run.name < $GITHUB_EVENT_PATH)" \!= "$NAME" ]
   then
-    echo wrong check
-    exit 1
+    outputResult wrong-check 1
```

Comment:
Does this work with `output_result: true`? The original code would `exit 1` here. Now we are setting `result` to something but falling through, I guess to the final `fi`?

Reply:
added an always exit step